spec/integration/create_a_document_spec.rb

```diff
@@ -12,45 +13,66 @@
     and_the_preview_creation_succeeded
   end
 
+  scenario "User create a document and the chosen base path is already in use" do
+    pending("@alex-ju looking into capybara tests")
+    when_i_click_on_create_a_document
+    and_i_choose_news
+    and_i_choose_a_press_release
+    and_i_fill_in_the_title_but_base_path_is_already_in_use
+    then_i_see_a_path_is_taken_error
+  end
+
   def when_i_click_on_create_a_document
     visit "/"
     click_on "New document"
   end
 
   def and_i_choose_news
-    choose "News"
+    choose "News", allow_label_click: true
```

Comment:
What's the `allow_label_click` for here? It doesn't seem to be necessary elsewhere...